microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/jdt/internal/openapi/MicroProfileOpenAPIConstants.java

```diff
@@ -21,4 +21,6 @@
 
 	public static String OPEN_API_CONFIG = "org.eclipse.microprofile.openapi.OASConfig";
 
+	public static final String RESPONSE_TYPE = "javax.ws.rs.core.Response";
```

Comment:
Move this constants in the existing jaxrsconstants and rename to RESPONSE_ANNOTATION

Reply:
It's a return type.

Reply:
Oops sorry for that, please just move this constant in the JaxRsConstants.